astropy/units/format/latex.py

```diff
@@ -48,7 +57,14 @@ def _format_unit_power(cls, unit, power=1):
         return name
 
     @classmethod
-    def _format_fraction(cls, scale, numerator, denominator, *, fraction="multiline"):
+    def _format_fraction(
+        cls,
+        scale: str,
+        numerator: str,
+        denominator: str,
+        *,
+        fraction: Literal[True, "inline", "multiline"] = "multiline",
+    ) -> str:
```

Comment:
This method has a different signature than the corresponding method in the parent class: https://github.com/astropy/astropy/blob/c68c3d29270b7f145583fe50273d04829e6a0f74/astropy/units/format/console.py#L64-L70
In the `Console` method `fraction` can be a positional argument, but in the `Latex` method it must be a keyword argument. We should update the `Latex` method so that `fraction` could be a positional argument here too, but code changes should be kept separate from adding type annotations.

Reply:
Agreed on not doing it here.
I kind of like it as a keyword-only here. Maybe we should update the other classes instead.

Reply:
I had a look at the `_format_fraction()` methods in the different formatter classes. Turns out the one in `Console` is the only one that can accept `fraction` as a positional argument, so that's the one we should edit.